apps/site/src/app/(home)/page.tsx

```diff
@@ -5,13 +5,18 @@ import Intro from "./sections/Intro";
 import GetInvolved from "./sections/GetInvolved";
 import Sponsors from "./sections/Sponsors";
 import FAQ from "./sections/FAQ";
+import Footer from "../../components/Footer/Footer";
 
 import styles from "./page.module.scss";
 
 const Home = () => {
 	return (
 		<div className={styles.home}>
 			<Landing />
+			{/* <Intro />
+			<Mentor />
+			<Sponsors />
+			<FAQ /> */}
```

Comment:
Chore: since these components have been replaced or won't be used, they can be removed completely.